lib/phlex/sgml/safe_value.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# @api private
+class Phlex::SGML::SafeValue
```

Comment:
Can we move this up to `Phlex::SafeValue`?

Reply:
I put this in SGML thinking, "is a safe SGML value have the same characteristics as a safe CSV (or other format) value?"

I can move it if you thought of that and still think it should be `Phlex::SafeValue`

Reply:
Oh, you're probably right. I was thinking it's considered safe only because it's marked as safe, so doesn't matter which output language it's for. But this is probably better.